web/src/stream_create.js

```diff
@@ -430,6 +452,7 @@ export function set_up_handlers() {
     $container.on("click", ".finalize_create_stream", (e) => {
         e.preventDefault();
         clear_error_display();
+        stream_settings_components.show_subs_pane.create_stream("configure_channel_settings");
```

Comment:
Why is this added? Due to this, if I select no subscribers in the "Subscribers" page and click on "Create" button, an error is shown there for not adding subscribers but error is not visible since the view is changed to settings panel.

Reply:
I added this code initially to handle a situation where the 'name' field might be empty. This would redirect the user to the configuration part. However, I've made some changes to the code, which now resolves this issue. Similarly, I've also addressed the same problem in the 'usergroup' section